src/main/java/com/ajou/hertz/domain/administrative_area/entity/Sido.java

```diff
@@ -0,0 +1,32 @@
+package com.ajou.hertz.domain.administrative_area.entity;
+
+import com.ajou.hertz.common.entity.TimeTrackedBaseEntity;
+import com.ajou.hertz.domain.administrative_area.constant.SidoList;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Table(name = "adminitrative_area_sido")
+public class Sido extends TimeTrackedBaseEntity{
+
+	@Id
+	@Column(nullable = false, updatable = false)
+	private Long id;
+
+	@Enumerated(EnumType.STRING)
+	@Column(nullable = false)
+	private SidoList name;
```

Comment:
```suggestion
	@Column(nullable = false)
	private String name;
```

항목이 많지 않고, 수정될 일이 많지 않을 것으로 보여 enum value를 사용하는 것도 좋아보입니다. 
다만, 시/군/구와 읍/면/동의 경우 항목이 많고 수정 소요를 고려하여 DB에서 값을 관리해야 할 것 같습니다. 그래서 시/도 역시 마찬가지로 단순 String으로 정의하고, DB에서 값을 관리하면 어떨까요?